src/scenes/Languages/Overview/LanguageOverview.tsx

```diff
@@ -0,0 +1,179 @@
+import {
+  Breadcrumbs,
+  Button,
+  Card,
+  CardContent,
+  Chip,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { CalendarToday } from '@material-ui/icons';
+import { Skeleton } from '@material-ui/lab';
+import { FC } from 'react';
+import * as React from 'react';
+import { useParams } from 'react-router-dom';
+import { UpdateLanguageInput } from '../../../api';
+import { Breadcrumb } from '../../../components/Breadcrumb';
+import { useDialog } from '../../../components/Dialog';
+import { DisplaySimpleProperty } from '../../../components/DisplaySimpleProperty';
+import {
+  useDateFormatter,
+  useNumberFormatter,
+} from '../../../components/Formatters';
+import { Sensitivity } from '../../../components/Sensitivity';
+import { EditLanguage } from '../Edit';
+import { useLanguageQuery } from './Language.generated';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  root: {
+    flex: 1,
+    overflowY: 'scroll',
+    padding: spacing(4),
+    '& > *': {
+      marginBottom: spacing(3),
+    },
+  },
+  container: {
+    display: 'flex',
+  },
+  edit: {
+    marginRight: spacing(1),
+  },
+  card: {
+    height: '210px',
```

Comment:
Heights like this should not be manually defined but based on the children and margin/padding instead.
```suggestion
```